ansible_roles/init/defaults/main.yml

```diff
@@ -0,0 +1 @@
+key_path: integration-tests/config/leappto_testing_key.pub
```

Comment:
Huh, Ansible is smart enough to try lookups relative to the repo root? Handy to know :)